package.json

```diff
@@ -95,7 +95,8 @@
     "**/typescript": "4.0.2",
     "**/unset-value": "^2.0.1",
     "**/jest-config": "npm:@amoo-miki/jest-config@27.5.1",
-    "**/jest-jasmine2": "npm:@amoo-miki/jest-jasmine2@27.5.1"
+    "**/jest-jasmine2": "npm:@amoo-miki/jest-jasmine2@27.5.1",
+    "**/http-cache-semantics": "^4.1.1"
```

Comment:
I think we don't need to resolve here. In the yarn.lock, we are fetching the latest patch version (^) of 4.1.0. Since the yarn.lock is not updated and just fetch the defined packags, all we need to do is remove this whole thing from yarn.lock:

```
http-cache-semantics@^4.0.0, http-cache-semantics@^4.1.0:
  version "4.1.0"
  resolved "https://registry.yarnpkg.com/http-cache-semantics/-/http-cache-semantics-4.1.0.tgz#49e91c5cbf36c9b94bcfcd71c23d5249ec74e390"
  integrity sha512-carPklcUh7ROWRK7Cv27RPtdhYhUsela/ue5/jKzjegVvXDqM2ILE9Q2BGn9JZJh1g87cp56su/FgQSzcWS8cQ==
```
Then do `yarn osd bootstrap` to let it fetch the latest patch release.  Then it should create something like 

```
http-cache-semantics@^4.0.0, http-cache-semantics@^4.1.0:
  version "4.1.1"
```

Reply:
I am following this https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/SECURITY.md. For sub-dependency, doc says to resolve by adding a new item in `resolutions` section. I see your point, I'll update the yarn.lock only to use `4.1.1`. But our documentation needs to be updated
@ananzh

Reply:
Good point. @zhongnansu could you create an issue for update SECURITY.md? Thanks